practico_08/App/App.py

```diff
@@ -0,0 +1,72 @@
+from flask import Flask, session, redirect, render_template, url_for, request
+from datetime import timedelta
+
+app = Flask(__name__)
+app.secret_key = "frro_soporte"
+app.permanent_session_lifetime = timedelta(minutes=3)
+
+
+@app.route("/")
+def home():
+    return go_to("log_in")
+
+
+@app.route('/<default_page>')
+def redirected_home(default_page):
+    return go_to("log_in")
+
+
+@app.route('/dashboard')
+def dashboard():
+    if logged():
+        return render_template('dashboard.html')
+    return go_to("log_in")
+
+
+@app.route('/sign_up', methods=["POST"])
+def sign_up():
+    delete_session_information()
+    if request.method == "POST":
+        return go_to("log_in")
+    return render_template('signUp.html')
+
+
+@app.route('/login', methods=["POST"])
+def log_in():
+    if logged():
+        return go_to("dashboard")
+    if request.method == "POST":
+        session.permanent = True
+        userName = request.form["userName"]
+        password = request.form["password"]
+        session["userName"] = userName
+        session["password"] = password
```

Comment:
Here we should add sqlalchemy validations, this will be added in the next step, probably tomorrow.